koku/masu/processor/azure/azure_report_processor.py

```diff
@@ -120,9 +121,11 @@ def _create_cost_entry_bill(self, row, report_db_accessor):
 
         report_date_range = utils.month_date_range(parser.parse(row_date))
         start_date, end_date = report_date_range.split("-")
+        start_date = start_date.replace(" +0000 UTC", "+0000")
+        end_date = end_date.replace(" +0000 UTC", "+0000")
```

Comment:
I'm not sure we need these here. Azure doesn't have the weird ` +0000 UTC` in its datetime strings.

Reply:
Sweet! I can totally take it out then. I saw that it didn't have it in our nise data, but I didn't know if the nise reports were exactly like azure reports so I added it to be safe. But if we are certain its not needed ill remove it.